### ISSUE TITLE
Complicated Language

### ISSUE THREAD

**Issue by hrittikhere** [harvester/docs#169]
Hey 
For this [page ](https://docs.harvesterhci.io/v1.0/networking/harvester-network/#management-network) the language seems complicated. Can we change it to `will change`? I can make a PR if that's okay 
![image](https://user-images.githubusercontent.com/67012359/170344668-dd367e6a-7f47-4103-ac07-074630582dbf.png)

**Comment by guangbochen** [comment]
It is a double negation, as in tradition, VM IPs are expected to remain unchanged after the VM reboot, cc @divya-mohan0209 any suggestions, thanks.

**Comment by hrittikhere** [comment]
Double negations are not that easy to read imo. If it's okay with most of the community a PR won't make sense. Looking forward to more information

**Comment by divya-mohan0209** [comment]
I agree with @hrittikhere here, @guangbochen. Double negations lower the readability. What we could do is rephrase the sentence to say 

..."and the management network IP will change after the VM reboot. This is non-typical behaviour that needs to be taken note of since VM IPs are expected to remain unchanged after a reboot."

This factors in both the things we're trying to cover by breaking the complex sentence into simpler phrases. What do you think?

**Comment by guangbochen** [comment]
Looks good to me, please file a PR to rephrase it, thanks guys.

**Comment by hrittikhere** [comment]
@divya-mohan0209 that sounds way better. Looking forward to the changes 😇
